x/evidence/keeper/infraction.go

```diff
@@ -73,27 +72,22 @@ func (k Keeper) handleEquivocationEvidence(ctx context.Context, evidence *types.
 	// Reject evidence if the double-sign is too old. Evidence is considered stale
 	// if the difference in time and number of blocks is greater than the allowed
 	// parameters defined.
-	resp, err := k.QueryRouterService.Invoke(ctx, &consensusv1.QueryParamsRequest{})
+
+	eviAgeBlocks, eviAgeDuration, _, err := k.consensusKeeper.EvidenceParams(ctx)
```

Comment:
Here, if EvidenceParams is nil, we error, while before we wouldn't

Reply:
it seems safe to skip even if not found. I was thinking of leaving it as returning nil.